authui/package.json

```diff
@@ -41,6 +41,7 @@
     "@sentry/tracing": "7.80.0",
     "@tabler/icons": "1.107.0",
     "axios": "1.6.1",
+    "countries-list": "^3.0.6",
```

Comment:
Use libphonenumber-js and cldr-localenames-full instead. It is based on Google's libphonenumber.
https://gitlab.com/catamphetamine/libphonenumber-js
https://github.com/unicode-org/cldr-json/tree/main/cldr-json/cldr-localenames-full

In particular, please

1. `import "libphonenumber-js/max"` to ensure we are using the full data set
2. Use `getCountries()` and `getCountryCallingCode()` to generate the country list.
3. Use `cldr-localenames-full` to get the localized locale names in the **CURRENT** language. The current language is `<html lang>`
4. Sort the list according to localized locale names.

And then follow this algorithm to get the emoji flag code point https://github.com/annexare/Countries/blob/main/packages/countries/src/getEmojiFlag.ts

Example

```
String.fromCodePoint(127462 - 'A'.charCodeAt(0) + 'H'.charCodeAt(0), 127462 - 'A'.charCodeAt(0) + 'K'.charCodeAt(0)) // 🇭🇰
```

Finally we can get rid of `countries-list`.